execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -231,6 +250,23 @@ impl LmdbWasmTestBuilder {
         Self::new_with_config(data_dir, Default::default())
     }
 
+    /// Creates new instance of builder and applies values only which allows the engine state to be
+    /// swapped with a new one, possibly after running genesis once and reusing existing database
+    /// (i.e. LMDB).
+    pub fn new_with_config_and_result<T: AsRef<OsStr> + ?Sized>(
+        data_dir: &T,
+        engine_config: EngineConfig,
+        result: &WasmTestResult<LmdbGlobalState>,
+    ) -> Self {
+        let mut builder = Self::new_with_config(data_dir, engine_config);
+        // Applies existing properties from gi
```

Comment:
I'm a little confused, what's gi?

Reply:
I'll be removing this method entirely, it snuck in during a rebase but it's not in dev.